app/src/organisms/ODD/RunningProtocol/ImageGalleryList/index.tsx

```diff
@@ -4,33 +4,44 @@ import { ListTable, StyledText } from '@opentrons/components'
 
 import { FloatingActionButton } from '/app/atoms/buttons'
 // eslint-disable-next-line opentrons/no-imports-across-applications
-import { useStubImagesInfo } from '/app/organisms/Desktop/Devices/ProtocolRun/ProtocolRunCamera/ImageGalleryContainer/hooks/useStubImagesInfo'
+import { useImageInfo } from '/app/organisms/Desktop/Devices/ProtocolRun/ProtocolRunCamera/ImageGalleryContainer/hooks/useImageInfo'
 import { GalleryListItem } from '/app/organisms/ODD/RunningProtocol/ImageGalleryList/GalleryListItem'
 import { ProtocolPlayPauseHeader } from '/app/organisms/ODD/RunningProtocol/shared/ProtocolPlayPauseHeader'
 
 import styles from './gallery.module.css'
 
 import type { RunStatus } from '@opentrons/api-client'
 // eslint-disable-next-line opentrons/no-imports-across-applications
-import type { UseStubImagesInfoResult } from '/app/organisms/Desktop/Devices/ProtocolRun/ProtocolRunCamera/ImageGalleryContainer/hooks/useStubImagesInfo'
+import type { UseImagesInfoItem } from '/app/organisms/Desktop/Devices/ProtocolRun/ProtocolRunCamera/ImageGalleryContainer/hooks/useImageInfo'
 
 export interface ImageGalleryListProps {
   runStatus: RunStatus | null
   onStop: () => void
   onTogglePlayPause: () => void
   protocolName: string | undefined
+  runId: string
+  protocolAnalysis: any
+  robotType: any
+  allRunDefs: any
```

Comment:
```suggestion
  runId: string
  protocolAnalysis: CompletedProtocolAnalysis
  robotType: RobotType
  allRunDefs: LabwareDefinition[]
```

We always try and define the types unless we're writing tests, in which case we often cheat. `any` effectively disables typechecking.